oio/xcute/common/backend.py

```diff
@@ -90,16 +90,17 @@ class XcuteBackend(RedisConnection):
         redis.call('ZADD', 'xcute:job:ids', 0, job_id);
         redis.call(
             'HSET', 'xcute:job:info:' .. job_id,
+            'job.id', job_id,
             'job.type', job_type,
             'job.status', 'WAITING',
             'job.request_pause', 'False',
             'tasks.all_sent', 'False',
             'tasks.sent', '0',
             'tasks.processed', '0',
-            'tasks.is_total_temp', 'True',
```

Comment:
I'd still like to have it. Especially if we also want to estimate the time the job is gonna take, how are we gonna do without it?.